src/index.html

```diff
@@ -5,8 +5,422 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#home"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#home" class="nav__link " >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#home" class="nav__link " >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="index.html"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#home" class="nav__link" >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#home" class="nav__link" >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#home" class="nav__link" >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#home" class="nav__link" >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main >
+      <ul class="catalog">
+        <li
+          data-qa="card"
+          class="card catalog__item"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__image"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <article class="card__review">
+            <ul class="stars">
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star stars__star--colored"></li>
+              <li class="stars__star"></li>
```

Comment:
please do the styling here using pseudoclasses, just as we did in the stars task

Reply:
not fixed